tutorials/fundamentals/compare-without-time.md

```diff
@@ -0,0 +1,30 @@
+If you want to compare two dates in JavaScript without using the time aspect, you should use
+the `toDateString()` method. It returns the date portion of the `Date` object as a string. From there,
+you can compare the two strings:
+
+```javascript
+const date1 = new Date().toDateString();
+const date2 = new Date(2000, 6, 25, 14, 39, 7).toDateString();
+
+date1 > date2 // true
+```
+
+## Handling Time Zones
+
+`toDateString()` does not take time zones into account. Using `toDateString()` without accounting for the coordinated universal time can create problems.
```

Comment:
Not quite correct, `toDateString()` converts the date to a string using your system's local time. So `new Date('14 Jun 2017 23:00:00 PDT');` is not on the same date as `new Date('14 Jun 2017 18:00:00 PDT');` if you convert them both to EDT.

Timezones are a bit of a headache. Reading this section, I think reworking it to use `toLocaleDateString()` is better: https://masteringjs.io/tutorials/fundamentals/date_format so you can show an example of taking two dates that are on the same date in one time zone, but not in another.